proposals/type-classes.md

```diff
@@ -281,24 +280,26 @@ Call site:
 fun addInts(): Int = add(1, 2)
 ```
 
-1. The compiler first looks at the function context where the invocation is happening. If a function argument matches the required instance for a typeclass, it uses that instance; e.g.:
+1. The compiler first looks at the function context where the invocation is happening. If a function argument matches the required instance for a type class, it uses that instance; e.g.:
 
-```kotlin
-fun <a> duplicate(a : A, with M: Monoid<A>): A = a.combine(a)
-```
+    ```kotlin
+    fun <a> duplicate(a : A, with M: Monoid<A>): A = a.combine(a)
+    ```
+
+    The invocation `a.combine(a)` requires a `Monoid<A>` and since one is passed as an argument to `duplicate`, it uses that one.
 
-The invocation `a.combine(a)` requires a `Monoid<A>` and since one is passed as an argument to `duplicate`, it uses that one.
+2. In case it fails, it inspects the following places, sequentially, until it is able to find a valid unique instance for the type class:
 
-2. In case it fails, it inspects the following places, sequentially, until it is able to find a valid unique instance for the typeclass:
-    a. The current package (where the invocation is taking place), as long as the `extension` is `internal`.
-    b. The companion object of the type parameter(s) in the type class (e.g. in `Monoid<A>`, it looks into `A`'s companion object).
-    c. The companion object of the type class.
-    d. The subpackages of the package where the type parameter(s) in the type class is defined.
-    e. The subpackages of the package where the type class is defined.
-3. If no matching implementation is found in either of these places fail to compile.
-4. If more than one matching implementation is found, fail to compile and indicate that there or conflicting instances.
+    * The current package (where the invocation is taking place), as long as the `extension` is `internal`.
+    * The companion object of the type parameter(s) in the type class (e.g. in `Monoid<A>`, it looks into `A`'s companion object).
+    * The companion object of the type class.
+    * The subpackages of the package where the type parameter(s) in the type class is defined.
+    * The subpackages of the package where the type class is defined.
+  
+3. If no matching implementation is found in either of these places then the code fails to compile.
```

Comment:
Replace the imperative mood here and below to keep with the mood used in the earlier points.